README.md

```diff
@@ -16,6 +16,295 @@ K8ssandra 1.x is configured, packaged, and deployed via Helm charts. Those Helm
 
 K8ssandra 2.x will be based on the this operator.
 
+One of the primary features of this operator is multi-cluster support which will facilitate multi-region Cassandra clusters.
+
+# Installing the operator
+There are a couple of options for installing the operator - build from source or remote installs via kustomize.
+
+**Note:** There are plans to add a Helm chart as well (TODO create and reference ticket here)
+
+**Note:** This section focuses on a single cluster install. See the multi-cluster section below for details on how to configure the operator for a multi-cluster install.
+
+## Remote Install
```

Comment:
Should we add a piece at the beginning here mentioning that you need to have a `kind` cluster already set up? For continuity through to the multi-cluster section we could even use the `setup-kind-multicluster.sh` script to set up a single cluster.

Reply:
kind is only required for development. While `setup-kind-multicluster.sh` can be generally used it is primarily intended for dev/testing.

Reply:
I just made another pass through the doc. I like the idea of more content around kind since that users can run it locally and since we provide `setup-kind-multicluster.sh`. I'll see what I can come up with.

Reply:
That's true. Some wording like `You must already have a k8s cluster running and kubectl pointing to it. If you wish to run locally you can use the `setup-kind-multicluster.sh` script to start a `kind` cluster` might be useful? Just thinking about absolute beginners here as we don't want to be fielding questions like "how do I start a kubernetes cluster" if we can avoid it.